src/path_converters.h

```diff
@@ -1037,18 +1037,19 @@ class Sketch
             // We want the "cursor" along the sine wave to move at a
             // random rate.
             double d_rand = m_rand.get_double();
-            double d_M_PI = 3.14159265358979323846;
             m_p += pow(m_randomness, d_rand * 2.0 - 1.0);
-            double r = sin(m_p / (m_length / (d_M_PI * 2.0))) * m_scale;
             double den = m_last_x - *x;
             double num = m_last_y - *y;
             double len = num * num + den * den;
             m_last_x = *x;
             m_last_y = *y;
             if (len != 0) {
+                const double d_M_PI = 3.14159265358979323846;
                 len = sqrt(len);
-                *x += r * num / len;
-                *y += r * -den / len;
+                double r = sin(m_p * (2.0 * d_M_PI) / m_length) * m_scale;
```

Comment:
Two minor changes since the review:

- Brackets to enable constant folding here.
- Change `+= x * -y` to `-= x * y` below.

At least both these give simplifications in the LLVM IR (maybe the backend will still provide the same code in the end though, but probably not for the first one due to FP aspects).